docs/contributing/versioning.md

```diff
@@ -41,32 +41,6 @@ This is to ensure the community can get involved with the decision.
 
 Make an active effort to involve the community, this might be in the form of presentations or meetings.
 
-## Deprecation
-
-Deprecation is the practice of communicating features in the library that should no longer be used and requires a user change behaviour in their application.
-
-Deprecating features in the library helps users migrate to the new code while still being able to run the older code.
-
-Note: Our users may not know what 'deprecation' means, so it's important to also clarify that we are no longer recommending the use of a feature.
-
-Example 1: Fixing a typo in a CSS class name.
-
-1. We discover the class name `.govuk-visually-hidden-focussable` includes the typo 'focussable'
-2. We raise a pull request that renames the class to `.govuk-visually-hidden-focusable` while keeping
-the previous class available.
-3. We add a comment to the source code that indicates this is deprecated, and raise an issue to remove it in a future breaking release.
-4. When releasing the change we include a clear summary that indicates what was the problem, what we've changed and how a user can migrate before the future breaking release.
-
-Sometimes it is not possible to deprecate code, this is OK but try to make this a last resort.
-
-### Sass doc deprecation annotations
-
-When deprecating Sass, you can use the [deprecated annotation](http://sassdoc.com/annotations/#deprecated).
-
-If there is an alias, for example if you are renaming something, you can use the [alias annotation](http://sassdoc.com/annotations/#alias).
-
-See [an example of deprecating a Sass mixin on GitHub](https://github.com/alphagov/govuk-frontend/blob/9424d87ed54764d2d8afe35d6e0077ee43d231e1/src/helpers/_grid.scss#L20-L26).
-
 ## Migration
```

Comment:
Should this be part of the new doc too?

Reply:
I don't think there's anything in there that's not said in the new doc in more detail, so have removed.